sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/analysis/AnalysisSuite.scala

```diff
@@ -831,4 +831,57 @@ class AnalysisSuite extends AnalysisTest with Matchers {
       }
     }
   }
+
+  test("SPARK-32131 Fix wrong column index when we have more than two columns" +
```

Comment:
nit: `SPARK-32131` -> `SPARK-32131:`

Reply:
Thanks maropu ,btw, I did not see the test build start to build, do you know why?